fda-model/metrics/ProjectPerformanceResults.cs

```diff
@@ -80,7 +83,42 @@ public double AssuranceOfAEP(double exceedanceProbability)
             double assuranceOfAEP = _aep.CDF(exceedanceProbability);
             return assuranceOfAEP;
         }
-
+        public bool ConditionalNonExceedanceProbabilityIsConverged()
+        {
+            //dont like this.
```

Comment:
would you prefer that we inject the recurrence interval at which to check for convergence?